### ISSUE TITLE
Visibility of the scrol bar in the visual cue editor in a black theme.

### ISSUE THREAD

**Issue by AmsterdamFM** [AzuraCast/AzuraCast#2725]
**Using Docker installation method**

Yes
**Host Operating System**

Ubuntu 16.04
**Describe the bug**

With the dark theme set the scroll bar is not visible in the visual cue editor after zooming in.
**To Reproduce**

1. Set the dark theme
2. go to a sound file.
3. Go to the visual cue editor.
4. Zoom in.
5. try scrolling trough the waveform.

**Expected behavior**

The knob for scrolling does not have the same color as the background
**Relevant Logs**

**Screenshots**

**Device(s):**
 - Device: Desktop  
 - OS: Windows 10
 - Browser: chrome 
 - Version: 81.0.4044.113 

**Additional context**

**Comment by CodeSteele** [comment]
![image](https://user-images.githubusercontent.com/31152835/80340957-930a7d00-882f-11ea-8817-fd8e70c96d67.png)

Had to drop in arrows because it's so hard to see.

**Comment by AmsterdamFM** [comment]
Thanks @CodeSteele ! Your picture also shows that the time markers are invisible. I have never used this feature on a light theme so i wondered why they were not there,

**Comment by CodeSteele** [comment]
Ah thanks for pointing that out too, yeah this is pretty easy to fix (a good thing for anyone with solid CSS knowledge to submit a PR for), but if no one gets around to it we'll be able to make the relevant CSS tweaks.

**Comment by AmsterdamFM** [comment]
Unfortunately I am not able to fix this as my CSS knowledge is extremely limited. 

I found another bug working with the visual cue editor. It has to do with big audio files. I will make a new issue for this as it is not related to this issue.

**Comment by SlvrEagle23** [comment]
![2020-05-05 23_52_32-(Development) Music Files - AzuraCast](https://user-images.githubusercontent.com/6744885/81139562-c8982000-8f2b-11ea-94c7-2fb78b217ce9.png)

Updated the styling in the latest update to improve the visibility of both the scrollbar and the time markers on the visual cue editor on the dark theme.

**Comment by AmsterdamFM** [comment]
Perfect! That solved it for me! Thank you for all the good work you do on the project!
